wscript

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import os
+
+from waflib.Configure import conf
+from waflib.Configure import ConfigurationContext
+
+APPNAME = 'waf-tools'
+VERSION = '2.34.0'
+
+@conf
+def load_external_waf_tool(ctx, name):
+    import inspect
+    this_file = inspect.getfile(inspect.currentframe())
+    path = os.path.join(os.path.dirname(this_file))
+    ctx.load([name], tooldir=[path])
+
+
+def resolve(ctx):
+
+    ctx.load_external_waf_tool('wurf_cxx_mkspec')
```

Comment:
Wouldn't a `cxt.resolve(...)` be able to work here? I never really liked the `load_external_waf_tool` thing. 

I don't know if it is possible but if top-level wscript in the `waf-tools` repo does the `recurse` step into the various .. I guess that should work? Which would be great because then `waf-tools` is just a dependency like any other.